index.html

```diff
@@ -47,8 +47,189 @@ <h1 class="h1">I'm Samson<br>Glad to see<br>you!</h1>
+                <div class="con">
+                    <a class="connect">
+                      LET'S CONNECT
+                    </a>
+                </div>
+                <ul class="social">
+                    <li><a><img src="images/twitter.svg" alt="twitter" class="icon"></a></li>
+                    <li><a><img src="images/Linkedin.svg" alt="Linkedin" class="icon"></a></li>
+                    <li><a><img src="images/Vector-3.svg" alt="Vector-3" class="icon"></a></li>
+                    <li><a><img src="images/Vector-2.svg" alt="Vector-2" class="icon"></a></li>
+                    <li><a><img src="images/Vector-4.svg" alt="Vector-4" class="icon"></a></li>
+                </ul>
+                <button><a>Get my resume</a></button>
+            </div>
+            <div class="footer-lang">
+                <div class="lang-title">
+                    <a>Languages</a>
+                    <a><img src="images/Icon - Chevron - Down - Larger.png" alt="vector"></a>
+                </div>
+                <div class="lang">
+                    <div class="javascript">
+                        <img src="images/Ellipse 1.png" alt="js">
+                        <a>javaScript</a>
+                    </div>
+                    <div class="html">
+                        <img src="images/Ellipse2.png" alt="html">
+                        <a>HTML</a>
+                    </div>
+                    <div class="css">
+                        <img src="images/Ellipse3.png" alt="css">
+                        <a>CSS</a>
+                    </div>
+                </div>
+                <div class="frame">
+                    <div class="framework">
+                        <a>Frameworks</a>
+                        <img src="images/Icon-arrow.png" alt="Icon-arrow">
+                    </div>
+                    <hr>
+                    <div class="skills">
+                        <a>Skills</a>
+                        <img src="images/Icon-arrow.png" alt="Icon-arrow">
+                    </div>
+                    <hr>
+                </div>
+            </div>
+        </div>
     </section>
 
+    <footer>
+        <div class="contact">
+            <div class="text-contact">
+                <h2>Contact me</h2>
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+                </p>
+                <div class="form-div">
+                    <form action="/action_page.php">
+                      <input type="text" id="fname" name="firstname" placeholder="Yeremias">
+                      <input type="text" id="lname" name="lastname" placeholder="natajayanj14@gmail.com">
+                      <textarea name="text" id="text" cols="30" rows="10"></textarea>
+                      <button><a>Get In Touch</a></button>
```

Comment:
:orange_circle: [OPTIONAL] I think it is a bad practice to nest a tag inside a button tag. Buttons are supposed to perform a specific action like submitting a form. When you nest one inside the other, it makes it confusing as to what action you want performed. Kindly remove keep the text and remove the a tag inside the button